art/estimators/certification/__init__.py

```diff
@@ -6,7 +6,6 @@
 from art.estimators.certification.randomized_smoothing.numpy import NumpyRandomizedSmoothing
 from art.estimators.certification.randomized_smoothing.tensorflow import TensorFlowV2RandomizedSmoothing
 from art.estimators.certification.randomized_smoothing.pytorch import PyTorchRandomizedSmoothing
-from art.estimators.certification.derandomized_smoothing.derandomized_smoothing import DeRandomizedSmoothingMixin
```

Comment:
I think we should keep one `DeRandomizedSmoothingMixin` for all frameworks even if it does not yet add any additional functionality it will serve to identify the type of the estimator.

Reply:
retained